Tests/Functional/Observation/All/main.swift

```diff
@@ -74,10 +78,10 @@ class Observation: XCTestCase {
         XCTAssertEqual(observer.finishedBundlePaths.count, 0)
 
         XCTFail("fail!")
-        XCTAssertEqual(observer.failureDescriptions, ["failed - fail!"])
+        XCTAssertEqual(observer.failureDescriptions, ["Observation.test_one : failed - fail!"])
```

Comment:
This is the test change I was referring to in the comment [above](https://github.com/apple/swift-corelibs-xctest/pull/86/files#r58315910).